core/server/data/schema/default-settings.json

```diff
@@ -82,7 +82,7 @@
             "defaultValue": "[{\"url\":\"\"}]"
         },
         "unsplash": {
-            "defaultValue": "[{\"applicationId\":\"\", \"isActive\": true}]"
+            "defaultValue": "{\"applicationId\":\"\", \"isActive\": false}"
```

Comment:
Do we still need the `applicationId` here?
From what i understood is, we either read the `applicationId` from the config if available or load the `applicationId` via client extensions.